release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -172,6 +172,11 @@ With this release, {product-title} updated to HAProxy 2.6.
 
 With this release, you can access the NMstate Operator and resources such as the `NodeNetworkState` (NNS), `NodeNetworkConfigurationPolicy` (NNCP), and `NodeNetworkConfigurationEnhancement` (NNCE) from the web console. In the *Administrator* perspective of the console from the *Networking* page you can access NNCP as well as NNCE from the *NodeNetworkConfigurationPolicy* page and NNS on the *NodeNetworkState* page. For more information about NMState resources and how to update them in the console, see xref:../networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.adoc#k8s-nmstate-updating-node-network-config[Updating node network configuration].
 
+[id="ocp-4-14-single-stack-support-nmstate"]
+==== Single stack IPv6 support for Kubernetes NMstate
+With this release, you can use kubernetes-nmstate operator in single-stack IPv6 clusters.
```

Comment:
In one line we have "single stack" and in the other "single-stack". Should be unified?

/lgtm

Reply:
I always write stackness with dash but it's only because "dual-stack" is used more often than "dual stack" according to google. I believe that both are correct, it's just a matter of convention. But even our docs are not consistent

Reply:
I will go with 'single-stack' then.